build.sh

```diff
@@ -1,18 +1,20 @@
 #!/bin/bash
-
 STYLE="\e[36;1m\e[1m"
 RESET="\e[0m"
 
 TYPE=${TYPE:-Release}
 
+rm -rf release
 mkdir -p release
-cd release
+cd release || exit
 
-echo -e "${STYLE}Compiling moonjit...${RESET}"
-pushd ../moonjit/src
-	make XCFLAGS+=-DLUAJIT_ENABLE_LUA52COMPAT -j${nproc}
-popd
+echo -e "${STYLE}Compiling (cmake) moonjit...${RESET}"
+pushd ../moonjit/src || exit
+make clean
+make XCFLAGS+=-DLUAJIT_ENABLE_LUA52COMPAT -j"${nproc}"
+popd || exit
 
-echo -e "${STYLE}Compiling RosaServer (${TYPE})...${RESET}"
-cmake -DCMAKE_BUILD_TYPE=${TYPE} ..
-make -j${nproc}
+echo -e "${STYLE}Compiling (cmake) RosaServer (${TYPE})...${RESET}"
```

Comment:
"Generating Makefiles", "Generating CMake files" or "Generating build files" would be better.

Reply:
Changed `Compiling (cmake) RosaServer` to `Generating build files with CMake`. The others made sense to me so I left them